src/roadmap/v1/lifecycle/rhel.py

```diff
@@ -171,7 +172,7 @@ async def get_relevant_systems(  # noqa: C901
         lifecycle_type = get_lifecycle_type(installed_products)
 
         # Collect system IDs by major version, minor version, and lifecycle type so we can return those in the response
-        system_id = result["id"]
+        system_id = (result["id"], result["display_name"])
```

Comment:
Same as above: make this an object.
```suggestion
        system_id = SystemInfo(id=result["id"], display_name=result["display_name"])
```